test/controllers/songs_controller_test.rb

```diff
@@ -1,19 +1,22 @@
-require 'test_helper'
+require "test_helper"
```

Comment:
At work and every other circle I've used single quotes, but I don't think there's a convention. I switched it to double quotes because the rest of the file was already using them.